This patch removes crates for Windows and IOS as GNU Guix does not support
building for these platforms.
--- glutin-0.28.0/Cargo.toml.orig	1973-11-29 22:33:09.000000000 +0100
+++ glutin-0.28.0/Cargo.toml.orig	2022-04-24 13:35:36.333502368 +0200
@@ -24,38 +24,9 @@
 lazy_static = "1.3"
 winit = { version = "0.26", default-features = false }
 
-[target.'cfg(target_os = "android")'.dependencies]
-android_glue = "0.2"
-glutin_egl_sys = { version = "0.1.5", path = "../glutin_egl_sys" }
-parking_lot = "0.11"
-
 [target.'cfg(target_os = "emscripten")'.dependencies]
 glutin_emscripten_sys = { version = "0.1.1", path = "../glutin_emscripten_sys" }
 
-[target.'cfg(any(target_os = "ios", target_os = "macos"))'.dependencies]
-objc = "0.2.6"
-glutin_gles2_sys = { version = "0.1.5", path = "../glutin_gles2_sys" }
-
-[target.'cfg(target_os = "macos")'.dependencies]
-cgl = "0.3"
-cocoa = "0.24"
-core-foundation = "0.9"
-
-[target.'cfg(target_os = "windows")'.dependencies.winapi]
-version = "0.3"
-features = [
-    "winnt",
-    "winuser",
-    "wingdi",
-    "libloaderapi",
-]
-
-[target.'cfg(target_os = "windows")'.dependencies]
-libloading = "0.7"
-glutin_wgl_sys = { version = "0.1.5", path = "../glutin_wgl_sys" }
-glutin_egl_sys = { version = "0.1.5", path = "../glutin_egl_sys" }
-parking_lot = "0.11"
-
 [target.'cfg(any(target_os = "linux", target_os = "freebsd", target_os = "dragonfly", target_os = "netbsd", target_os = "openbsd"))'.dependencies]
 osmesa-sys = "0.1"
 wayland-client = { version = "0.29", features = ["dlopen"], optional = true }
